### ISSUE TITLE
Stack overflow crash

### ISSUE THREAD

**Issue by bravoso** [Asek3/Oculus#44]
### Minecraft Version

1.16.5

### Iris Version

oculus-1.16.5-1.2.2.jar

### Sodium Version

rubidium-1.16.5-0.2.4oculus.jar

### Operating System

Windows 11

### What is your GPU?

GTX 1070 Ti

### Java Version

Java 8

### Reproduction Steps

Loading a game with complementary Shaders. If you're close to spawn, it crashes. If you're away from spawn, it works normally.

### Crash Report file and latest.log

```shell
https://gist.github.com/bravoso/a291a3cd9b6c675616a7732894ba6ee3
```

### Additional context

[latest.log](https://github.com/Asek3/Oculus/files/8459663/latest.log)
[crash-2022-04-10_15.13.57-client.txt](https://github.com/Asek3/Oculus/files/8459664/crash-2022-04-10_15.13.57-client.txt)
[crash-2022-04-10_14.36.07-client.txt](https://github.com/Asek3/Oculus/files/8459665/crash-2022-04-10_14.36.07-client.txt)
[crash-2022-04-10_14.48.24-client.txt](https://github.com/Asek3/Oculus/files/8459666/crash-2022-04-10_14.48.24-client.txt)
[crash-2022-04-10_14.56.21-client.txt](https://github.com/Asek3/Oculus/files/8459667/crash-2022-04-10_14.56.21-client.txt)

**Comment by bravoso** [comment]
I've tested on the same enviroment with SEUS Renewed, and the problem did not happen. So it's probably an issue related to rendering the Complementary shaders

**Comment by KarinMiku** [comment]
yep i can confirm, but i didnt know that it was related to the spawn, i thought it was just a crash that could happen randomly lol, but yea complemetary shaders arent working at all

**Comment by Asek3** [comment]
So, obviously fixed in 1.2.5
